.gitattributes

```diff
@@ -0,0 +1,4 @@
+doc/* linguist-documentation
+src/test/* linguist-vendored
```

Comment:
Sure you also wanted to add this file?

Reply:
Do you want your repo to be a HTML repo?